src/openforms/registrations/contrib/zgw_apis/options.py

```diff
@@ -245,6 +257,41 @@ def _validate_against_catalogi_api(attrs: RegistrationOptions) -> None:
         )
 
 
+def _validate_against_objects_api_group(attrs: RegistrationOptions) -> None:
+    """
+    Validate the configuration options against the specified objects API group.
+
+    1. An `objects_api_group` must be specified
+    2. The specified `objecttype` must have the same base URL as the defined `objects_api_group.objecttypes_service`
+    """
+    assert "objecttype" in attrs
+
+    objects_api_group = attrs.get("objects_api_group")
+    if not objects_api_group:
+        raise serializers.ValidationError(
+            {
+                "objects_api_group": _(
+                    "Objects API group must be specified if an objecttype is specified."
+                )
+            },
+            code="invalid",
```

Comment:
```suggestion
            code="required",
```
or which code does DRF use for missing/blank/empty values?

Reply:
this is the last thing to pick up, barring any CI nopes